main/customLogger.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"runtime"
+	"strings"
+	"time"
+)
+
+var (
+	infoOperation    = "INFO"
+	warningOperation = "WARNING"
+	errorOperation   = "ERROR"
+)
+
+type customInfoLogger struct {
+	infoLogger *log.Logger
+}
+
+func (iLogger *customInfoLogger) Print(v ...interface{}) {
```

Comment:
There's a lot of code-duplication. Can you parameterize this? Make this a function instead or something?